config/initializers/sidekiq.rb

```diff
@@ -22,4 +22,7 @@
   config.client_middleware do |chain|
     chain.add Sidekiq::Instrument::ClientMiddleware
   end
+
+  # Remove the default error handlers, leave in the injected Raven handler 
+  config.error_handlers.delete_if { |handler| !handler.is_a?(Raven::Sidekiq) }
```

Comment:
Do we want to remove everything except Sentry, or just the logger? We could remove just the logger by deleting the error handler that `is_a?(Sidekiq::ExceptionHandler::Logger)`